raft.c

```diff
@@ -149,11 +152,13 @@ static void executeRaftRedisCommandArray(RaftRedisCommandArray *array,
             continue;
         }
 
+        uint64_t begin_rmcall = timeMonotonicNanos();
```

Comment:
I see that inside RedisModule_Call(), it measures time passed. Maybe we avoid doing that here?